### DISCUSSION TITLE
Floating keyboard

### DISCUSSION THREAD

**Discussion by erchanda** [AnySoftKeyboard/AnySoftKeyboard#3427] [Q&A]
I just downloaded ASK and went straight to customizing. Looks good! However I haven't been able to find an option to resize the keyboard. The closest thing is the option to have the keyboard squeezed to the right : it's the right size, but the whole screen to the left of the keys is black,  so half the screen is blocked! Not good!

I'm working on a large Android tablet 12" so I'd like to be able to have a floating keyboard of a small enough size that I can use gesture typing comfortably. Is that available? If not, please add that feature ASAP!

**Comment by hfm158** [comment]
See also 
https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/2488
and
https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1952
